web-ui/src/components/guides/GuidesPanel.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from "react";
+import { AppContext } from "../../context/AppContext";
+import "./GuidesPanel.css";
+import GuideLink from "./GuideLink"
+
+const GuidesPanel = () => {
+    const { state } = useContext(AppContext);
+    const isPdl = state.user.role === "pdl";
```

Comment:
with what we have right now, though I think it may change, this would be state.userDate.role

Reply:
I based this on the defaultProfile field on AppContext.js. I find it very plausible that this isn't how it's going to work in reality, bu the guidance I got is that this profile stuff isn't fully implemented at all so whatever I put here will probably have to change.